crates/re_log/src/setup.rs

```diff
@@ -59,6 +59,14 @@ pub fn setup_native_logging() {
 
 #[cfg(target_arch = "wasm32")]
 pub fn setup_web_logging() {
+    use std::sync::atomic::{AtomicBool, Ordering};
+
+    static LOG_INIT: AtomicBool = AtomicBool::new(false);
+    if LOG_INIT.load(Ordering::SeqCst) {
+        return;
+    }
+    LOG_INIT.store(true, Ordering::SeqCst);
```

Comment:
Calling `new WebViewer` twice would cause a panic here, because the logger would be set by the previous call. Not sure if this is the best way to fix this, as a panic during initialization will cause the logger to not work anymore.